packages/zoe/src/internal-types.js

```diff
@@ -152,7 +166,7 @@
  * @param {ProposalRecord} proposal
  * @param {ExitObj} exitObj
  * @param {SeatHandle} seatHandle
- * @returns {{userSeat: UserSeat, notifier: Notifier<Allocation>}}
+ * @returns {{userSeat: UserSeat}}
```

Comment:
If this now only needs to return one thing, should it stop being a kit maker and just return that one thing?
```suggestion
 * @returns {UserSeat}
```

Reply:
It's not an external API, so yeah, that's a cheap update.